routes/pagination.go

```diff
@@ -1,19 +1,58 @@
 package routes
 
 import (
+	"github.com/gin-gonic/gin"
 	"gorm.io/gorm"
+	"rest-api/util"
+	"strconv"
 )
 
 type PaginatedResponse struct {
-	Page      uint
-	PageSize  uint
-	TotalSize uint
+	Page      int
```

Comment:
Is this intended? Is there a negative page number?

Reply:
gorm accepts int, this way we don't need to cast a bunch. openapi docs will only show a generic `integer` anyway, so I see no issue

Reply:
Where is this a gorm related topic?

Reply:
https://github.com/helpwave/rest-api/pull/75/files#diff-9b2ae1c4edb52df1e3a29779c57a265566dd0955af2867052761a7e37b038831R23